howso/typing.amlg

```diff
@@ -0,0 +1,159 @@
+(null
+
+    #!CustomTypes
```

Comment:
This should be lowercase #!customTypes since it's not a method that is going to be called, but a datastore/lookup

Reply:
will do